scripts/scoring_test.py

```diff
@@ -0,0 +1,29 @@
+import requests
+from bs4 import BeautifulSoup
+import numpy as np
+
+DEBUG_URL = 'http://127.0.0.1:5000'
+
+def get_word_score(language1, language2, spelling):
+    response = requests.post(f'{DEBUG_URL}/translitteration/', data = {'language1': language1, 'language2': language2, 'spelling': spelling})
+    data = response.text
+    soup = BeautifulSoup(data, "html5lib")
+    score = soup.find(id="harmonic_mean").getText()
+    return(int(score[:-2]))
+
+def get_score(language1, language2, text_file):
+    with open(text_file) as text_file:
+        all_scores = []
+        not_found = 0
+        for word in text_file:
+            try:
+                score = get_word_score(language1, language2, word)
+                all_scores += [score]
```

Comment:
`all_scores.append(score)`  it's cleaner since it does not do a list concatenation, but merely appends to the end of the list.